scripts/dependencies.json

```diff
@@ -8,68 +8,68 @@
   {
     "groupId": "org.sonarsource.slang",
     "artifactId": "sonar-go-plugin",
-    "version": "1.17.1.145",
+    "version": "1.18.0.240",
     "output": "analyzers/sonargo.jar"
   },
   {
     "groupId": "org.sonarsource.javascript",
     "artifactId": "sonar-javascript-plugin",
-    "version": "10.16.0.27621",
+    "version": "10.17.0.28100",
     "output": "analyzers/sonarjs.jar"
   },
   {
     "groupId": "org.sonarsource.java",
     "artifactId": "sonar-java-plugin",
-    "version": "8.5.0.37199",
+    "version": "8.6.0.37351",
     "output": "analyzers/sonarjava.jar"
   },
   {
     "groupId": "org.sonarsource.java",
     "artifactId": "sonar-java-symbolic-execution-plugin",
-    "version": "8.5.0.37199",
+    "version": "8.6.0.37351",
     "output": "analyzers/sonarjavasymbolicexecution.jar"
   },
   {
     "groupId": "org.sonarsource.php",
     "artifactId": "sonar-php-plugin",
-    "version": "3.38.0.12239",
+    "version": "3.40.0.12590",
     "output": "analyzers/sonarphp.jar"
   },
   {
     "groupId": "org.sonarsource.python",
     "artifactId": "sonar-python-plugin",
-    "version": "4.23.0.17664",
+    "version": "4.24.0.18631",
     "output": "analyzers/sonarpython.jar"
   },
   {
     "groupId": "org.sonarsource.html",
     "artifactId": "sonar-html-plugin",
-    "version": "3.16.0.5274",
+    "version": "3.17.0.5548",
     "output": "analyzers/sonarhtml.jar"
   },
   {
     "groupId": "org.sonarsource.xml",
     "artifactId": "sonar-xml-plugin",
-    "version": "2.11.0.5671",
+    "version": "2.12.0.5749",
     "output": "analyzers/sonarxml.jar"
   },
   {
     "groupId": "org.sonarsource.text",
     "artifactId": "sonar-text-plugin",
-    "version": "2.18.0.4812",
+    "version": "2.18.0.4866",
```

Comment:
Please wait with the update of the SonarText analyzer, they found a bug and will release a patch